pkg/apiclient/http1-client.go

```diff
@@ -18,6 +18,10 @@ func (h httpClient) NewArchivedWorkflowServiceClient() (workflowarchivepkg.Archi
 	return http1.ArchivedWorkflowsServiceClient(h), nil
 }
 
+func (a httpClient) NewOfflineWorkflowServiceClient() OfflineWorkflowServiceClient {
```

Comment:
I think we should have another implementation of `apiclient.Client` named `OfflineClient` and than `apiclient.NewClientFromOpts` can take `Opts{Offline: true}`.